tests/main/snap-quota-journal/task.yaml

```diff
@@ -0,0 +1,100 @@
+summary: Functional test for journal quota-related snap commands.
+
+details: |
+  Functional test for snap journal quota group commands ensuring that they are 
+  effective in practice.
+
+# these systems do not support journal quota groups due to their old systemd versions.
+# requires systemd v245+
+systems:
+  - -amazon-linux-2-*
+  - -centos-7-*
+  - -centos-8-*
+  - -debian-10-64
+  - -ubuntu-14.04-*
+  - -ubuntu-16.04-*
+  - -ubuntu-18.04-*
+  - -ubuntu-core-16-*
+  - -ubuntu-core-18-*
+
+prepare: |
+  snap pack test-snapd-journal-quota
+  snap install --dangerous test-snapd-journal-quota_1.0_all.snap
+  snap set system experimental.quota-groups=true
+  tests.cleanup defer snap unset system experimental.quota-groups
+
+execute: |
+  echo "Starting service and verifying that log messages are sent to default journal namespace"
+  snap start test-snapd-journal-quota.logger
+
+  # Now ask for logs and lets see if we can see messages from 'LogDaemon'
+  # The format of the log messages are 'LogDaemon: message number: X'
+  retry -n 5 sh -c "$TESTSTOOLS/journal-state get-log | MATCH LogDaemon"
+
+  # Create a journal quota group and set arbitrary limits, we will then verify these are written
+  # correctly for journald
+  echo "Create a group with the logger snap in it"
+  snap set-quota group-one --journal-size=16MB --journal-rate-limit=100/10ms test-snapd-journal-quota
+
+  echo "The systemd slice should be active now"
+  sliceName="snap.$(systemd-escape --path group-one).slice"
+  systemctl show --property=ActiveState "$sliceName" | MATCH "ActiveState=active"
```

Comment:
is it the same than using 'systemctl is-active "$sliceName"' ?

Reply:
Ah cool, thanks for this!